src/video/kmsdrm/SDL_kmsdrmvideo.c

```diff
@@ -55,15 +55,17 @@
 #ifdef __OpenBSD__
 static SDL_bool openbsd69orgreater = SDL_FALSE;
 #define KMSDRM_DRI_PATH openbsd69orgreater ? "/dev/dri/" : "/dev/"
+#define KMSDRM_DRI_PATHSIZE    openbsd69orgreater ? 9 : 5
```

Comment:
Suggest: `SDL_strlen(KMSDRM_DRI_PATH)` - this isn't in a performance critical path, and it's more future proof if someone adjusts KMSDRM_DRI_PATH and doesn't notice that the size needs to be adjusted as well.

Reply:
It also means the size definitions can be outside the platform specific ifdef, which is nice.

Reply:
Everything is runtime now, tested on Linux